src/Disks/DiskDecorator.h

```diff
@@ -57,6 +57,7 @@ class DiskDecorator : public IDisk
     void removeSharedFiles(const RemoveBatchRequest & files, bool keep_all_batch_data, const NameSet & file_names_remove_metadata_only) override;
     void setLastModified(const String & path, const Poco::Timestamp & timestamp) override;
     Poco::Timestamp getLastModified(const String & path) override;
+    time_t getLastChanged(const String & path) override;
```

Comment:
Now it looks quite weird (comparing with previous method) :)
Maybe add a comment like "We hate `Poco::Timestamp` but it is so tricky and powerful so we cannot get rid of it at once"